share/transformers/org_swbiodiversity.py

```diff
@@ -125,6 +143,15 @@ def unwrap_data(self, input_data):
             collection_statistics = start.find_all_next('li')
             collection_statistics = list(map(self.extract_text, collection_statistics))
             data['collection-statistics'] = self.process_collection_stat(collection_statistics)
+
+            link = record.find_next('div', style="margin:3px;")
```

Comment:
Instead of all this (and instead of `identifiers = ...` above), you could yield the full URL as the identifier [from the harvester](https://github.com/sheriefvt/SHARE/blob/e7eb3f0b8aea33d88ff06aba71a2c35032be5319/share/harvesters/org_swbiodiversity.py#L64) (`yield collection_page.url, str(record)`). It will automatically be added as an identifier.